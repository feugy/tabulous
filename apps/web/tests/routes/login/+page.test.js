import LoginPage from '@src/routes/login/+page.svelte'
import { render } from '@testing-library/svelte'
import html from 'svelte-htm'
import { describe, expect, it, vi } from 'vitest'

vi.mock('$app/stores', () => {
  const { BehaviorSubject } = require('rxjs')
  return {
    page: new BehaviorSubject({
      url: new URL('http://localhost/login'),
      route: { id: '/login' }
    })
  }
})

describe('/login route', () => {
  it('displays all options', async () => {
    const { container } = render(html`<${LoginPage} />`)
    expect(container).toMatchSnapshot()
  })

  it('expands login panel on error', async () => {
    const { container } = render(
      html`<${LoginPage} form=${{ message: 'wrong password' }} />`
    )
    expect(container).toMatchSnapshot()
  })
})
