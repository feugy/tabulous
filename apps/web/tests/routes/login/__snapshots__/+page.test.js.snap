// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`/login route > displays all options 1`] = `
<body>
  <div>
     
    <main
      class="s-BUJUVtHZo5xi"
    >
      <i
        class="s-BUJUVtHZo5xi"
        id="top"
      />
       
      <header
        class="s-c1MG4BWNDc59"
      >
        <nav
          class="s-c1MG4BWNDc59"
        >
          <button
            class="s-c1MG4BWNDc59"
          >
            <svg
              height="27.035763"
              version="1.1"
              viewBox="0 0 37.126423 27.035763"
              width="37.126423"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:svg="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(-5.7081121,-5.3321023)"
              >
                <path
                  d="m 25.120364,19.713484 a 16.171131,6.3284572 0.15697335 0 0 -10.950753,1.507401 c 0.140483,0.350402 0.240812,0.598413 0.240812,0.598413 l -0.787548,-0.409277 a 16.171131,6.3284572 0.15697335 0 0 -1.884123,0.829407 c 0.463249,0.204263 0.790513,0.438583 0.805119,0.730704 0.03586,0.717292 -1.395039,1.030311 -2.9124841,0.762228 a 16.171131,6.3284572 0.15697335 0 0 -1.0185424,1.698604 16.171131,6.3284572 0.15697335 0 0 14.6461345,6.907589 16.171131,6.3284572 0.15697335 0 0 17.564302,-5.691126 16.171131,6.3284572 0.15697335 0 0 0.0016,-1.133781 c -1.581229,1.267596 -4.142285,2.250232 -4.714441,0.95188 -0.524486,-1.190168 2.444087,-1.664571 3.564123,-2.796728 a 16.171131,6.3284572 0.15697335 0 0 -5.535579,-2.750737 c -0.157951,1.104143 0.2541,3.296302 -1.137915,3.298507 -0.816889,0.0013 -1.122444,-2.159577 -0.807186,-3.771346 a 16.171131,6.3284572 0.15697335 0 0 -6.017204,-0.705383 16.171131,6.3284572 0.15697335 0 0 -1.056266,-0.02635 z m -0.912606,1.601969 h 1.580782 v 1.623674 h -1.580782 z m -5.183146,0.960148 1.791105,0.735873 -1.012858,0.817521 -1.791105,-0.735873 z m 11.035502,0.810287 1.012858,0.817521 -1.791105,0.735872 -1.012858,-0.817521 z m -11.185364,0.681095 1.791106,0.735872 -1.013375,0.817522 -1.791105,-0.735873 z m 15.300875,0.373621 1.012858,0.817521 -1.791105,0.735872 -1.012858,-0.817521 z m -17.849039,0.125057 1.791105,0.735872 -1.012858,0.817521 -1.791106,-0.735872 z"
                  style="font-variation-settings: normal; fill: #000000; fill-opacity: 1; fill-rule: evenodd; stroke: none; stroke-width: 0.634999; stroke-linecap: round; stroke-linejoin: miter; stroke-miterlimit: 4; stroke-dasharray: none; stroke-dashoffset: 0; stroke-opacity: 0.5117; stop-color: rgb(0, 0, 0);"
                />
                <path
                  d="m 35.550623,21.31134 0.668087,-2.021097 c 0.159717,-0.483175 -1.429238,0.453831 -1.991196,1.166561 -0.54224,0.687723 0.350734,3.781871 -1.30307,3.78449 -1.073283,0.0017 -1.264756,-3.730088 -0.323839,-5.043292 0.553159,-0.772023 3.068597,-2.275848 4.221601,-2.288246 1.590592,-0.0171 4.536182,1.713781 5.305128,2.835228 0.562386,0.820194 0.221571,3.922708 -0.437095,4.841158 -1.149572,1.602976 -4.944053,3.520061 -5.656223,1.903992 -0.561409,-1.273959 2.879328,-1.727923 3.771314,-3.046188 0.703797,-1.040142 0.517619,-2.429842 0.517619,-2.429842 m -2.654858,-7.866815 a 2.3234062,2.8584074 14.77332 0 0 1.751252,3.171591 2.3234062,2.8584074 14.77332 0 0 2.844305,-2.433408 2.3234062,2.8584074 14.77332 0 0 -1.751253,-3.171591 2.3234062,2.8584074 14.77332 0 0 -2.844304,2.433408 z"
                  style="font-variation-settings: normal; fill: none; fill-opacity: 1; stroke: #000000; stroke-width: 0.846663; stroke-linecap: round; stroke-linejoin: miter; stroke-miterlimit: 4; stroke-dasharray: none; stroke-dashoffset: 0; stroke-opacity: 1; stop-color: rgb(0, 0, 0);"
                />
                <path
                  d="m 21.69336,20.588834 c 0,0 0.02678,-2.677038 -0.206019,-3.091041 -0.776379,-1.380737 -2.599184,-4.025216 -2.613438,-5.966523 -0.01639,-2.2322799 1.752164,-6.4643449 2.726857,-5.6885019 1.078456,0.858437 -1.085657,4.5175159 -0.381412,6.1633349 0.418802,0.978742 1.010211,2.291873 1.83974,2.745834 1.018025,0.557116 3.010225,0.631654 4.047692,0.111544 1.21019,-0.606701 1.861782,-1.447969 2.336439,-2.879006 0.427503,-1.288873 -1.154588,-4.9427149 -0.07921,-5.3658779 1.462782,-0.575606 2.312798,3.6986019 2.203533,5.6037499 -0.103141,1.798372 -2.31399,4.015614 -2.819254,5.217566 -0.341408,0.812161 -0.156693,3.467564 -0.156693,3.467564 M 27.33896,11.429663 a 2.033196,2.750803 1.8373958 0 1 -2.29222,2.410331 2.033196,2.750803 1.8373958 0 1 -1.737975,-3.052389 2.033196,2.750803 1.8373958 0 1 2.29222,-2.4103309 2.033196,2.750803 1.8373958 0 1 1.737975,3.0523889 z"
                  style="font-variation-settings: normal; fill: none; fill-opacity: 1; stroke: #000000; stroke-width: 0.846663; stroke-linecap: round; stroke-linejoin: miter; stroke-miterlimit: 4; stroke-dasharray: none; stroke-dashoffset: 0; stroke-opacity: 1; stop-color: rgb(0, 0, 0);"
                />
                <path
                  d="m 14.333877,21.844254 c 0,0 -1.220706,-3.003311 -1.32075,-3.465503 -0.05842,-0.269875 1.287063,1.131397 2.623366,0.903835 1.49769,-0.255045 3.519874,-4.235044 1.763283,-4.584697 -0.664466,-0.132263 -1.50618,2.482816 -2.146704,2.604076 -0.595067,0.112655 -2.319707,-2.019375 -3.525543,-2.212273 -2.186247,-0.349735 -3.979353,0.802561 -4.169399,1.046062 -1.192202,1.527537 -1.886942,4.551923 -1.053854,5.856331 1.452582,2.27438 6.026744,2.278506 5.962967,1.002872 -0.0501,-1.002024 -3.762944,-1.331489 -4.117637,-2.056564 -0.192935,-0.394403 0.201973,-2.099557 0.201973,-2.099557 m 2.667282,-7.575614 A 2.8582202,2.3235584 84.877672 0 1 8.9625898,14.098673 2.8582202,2.3235584 84.877672 0 1 6.5644114,11.225541 2.8582202,2.3235584 84.877672 0 1 8.8206822,8.3900889 2.8582202,2.3235584 84.877672 0 1 11.218861,11.263222 Z"
                  style="fill: none; stroke: #000000; stroke-width: 0.846663; stroke-linecap: round; stroke-linejoin: miter; stroke-dasharray: none; stroke-opacity: 1;"
                />
              </g>
            </svg>
            <!--&lt;Tabulous-logo.svg.rollup-plugin&gt;-->
          </button>
           
          <div
            class="s-c1MG4BWNDc59"
          >
            <h1
              class="s-BUJUVtHZo5xi"
            >
              Connexion
            </h1>
          </div>
           
          <span
            class="nav s-c1MG4BWNDc59"
          />
        </nav>
         
        <div
          class="s-c1MG4BWNDc59"
        />
      </header>
      <!--&lt;Header&gt;-->
       
      <section
        class="s-BUJUVtHZo5xi"
      >
        <span
          class="s-j2LTsWktqK_4"
        >
          <div
            class="pane s-j2LTsWktqK_4"
          >
            <h2
              class="s-j2LTsWktqK_4"
            >
              Bienvenue !
            </h2>
            
             
            <div
              class="container s-OEREYjlWzGh2"
            >
              <button
                class="has-text s-NMyTiX1zi6rz"
              >
                <span
                  class="icon s-NMyTiX1zi6rz"
                >
                  <svg
                    height="20"
                    slot="icon"
                    viewBox="0 0 14 14"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M7 .175c-3.872 0-7 3.128-7 7 0 3.084 2.013 5.71 4.79 6.65.35.066.482-.153.482-.328v-1.181c-1.947.415-2.363-.941-2.363-.941-.328-.81-.787-1.028-.787-1.028-.634-.438.044-.416.044-.416.7.044 1.071.722 1.071.722.635 1.072 1.641.766 2.035.59.066-.459.24-.765.437-.94-1.553-.175-3.193-.787-3.193-3.456 0-.766.262-1.378.721-1.881-.065-.175-.306-.897.066-1.86 0 0 .59-.197 1.925.722a6.754 6.754 0 0 1 1.75-.24c.59 0 1.203.087 1.75.24 1.335-.897 1.925-.722 1.925-.722.372.963.131 1.685.066 1.86.46.48.722 1.115.722 1.88 0 2.691-1.641 3.282-3.194 3.457.24.219.481.634.481 1.29v1.926c0 .197.131.415.481.328C11.988 12.884 14 10.259 14 7.175c0-3.872-3.128-7-7-7z"
                      fill="currentColor"
                    />
                  </svg>
                  <!--&lt;Github-logo.svg.rollup-plugin&gt;-->
                </span>
                
                <span
                  class="text s-NMyTiX1zi6rz"
                >
                  Continuer avec Github
                </span>
                
                
              </button>
              <!--&lt;Button&gt;-->
               
              <button
                class="has-text s-NMyTiX1zi6rz"
              >
                <span
                  class="icon s-NMyTiX1zi6rz"
                >
                  <svg
                    height="18"
                    slot="icon"
                    width="17.64"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M8.991 10.71V7.362h8.424c.126.567.225 1.098.225 1.845C17.64 14.346 14.193 18 9 18c-4.968 0-9-4.032-9-9s4.032-9 9-9c2.43 0 4.464.891 6.021 2.349l-2.556 2.484C11.817 4.221 10.683 3.501 9 3.501c-2.979 0-5.409 2.475-5.409 5.508S6.021 14.517 9 14.517c3.447 0 4.716-2.385 4.95-3.798H8.991Z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                  </svg>
                  <!--&lt;Google-logo.svg.rollup-plugin&gt;-->
                </span>
                
                <span
                  class="text s-NMyTiX1zi6rz"
                >
                  Continuer avec Google
                </span>
                
                
              </button>
              <!--&lt;Button&gt;-->
               
              <details
                class="s-OEREYjlWzGh2"
              >
                <summary
                  class="s-OEREYjlWzGh2"
                  title="password-toggle"
                >
                  <span
                    class="material-icons s-OEREYjlWzGh2"
                  >
                    arrow_forward
                  </span>
                  Continuer avec un mot de passe
                </summary>
                 
                <form
                  class="s-OEREYjlWzGh2"
                  method="POST"
                >
                  <div
                    class="row s-OEREYjlWzGh2"
                  >
                    <fieldset
                      class="s-Fe6WNbR-55zp"
                    >
                      <legend
                        class="s-Fe6WNbR-55zp"
                      >
                        Identifiant
                      </legend>
                       
                      <input
                        class="s-Fe6WNbR-55zp"
                        name="id"
                      />
                    </fieldset>
                    <!--&lt;Input&gt;-->
                  </div>
                   
                  <div
                    class="row s-OEREYjlWzGh2"
                  >
                    <fieldset
                      class="s-Fe6WNbR-55zp"
                    >
                      <legend
                        class="s-Fe6WNbR-55zp"
                      >
                        Mot de passe
                      </legend>
                       
                      <input
                        class="s-Fe6WNbR-55zp"
                        data-testid="password"
                        name="password"
                        type="password"
                      />
                    </fieldset>
                    <!--&lt;Input&gt;-->
                  </div>
                   
                   
                  <div
                    class="actions s-OEREYjlWzGh2"
                  >
                    <button
                      class="primary has-text s-NMyTiX1zi6rz"
                      disabled=""
                      type="submit"
                    >
                      <span
                        class="material-icons s-NMyTiX1zi6rz"
                      >
                        emoji_people
                      </span>
                      
                      <span
                        class="text s-NMyTiX1zi6rz"
                      >
                        Entrer
                      </span>
                      
                      
                    </button>
                    <!--&lt;Button&gt;-->
                  </div>
                   
                  <input
                    class="s-OEREYjlWzGh2"
                    name="redirect"
                    type="hidden"
                    value=""
                  />
                </form>
              </details>
            </div>
          </div>
        </span>
        <!--&lt;Pane&gt;-->
        <!--&lt;Form&gt;-->
      </section>
       
      <footer
        class="s-_Wq18H0k1xjH"
      >
        <section
          class="s-_Wq18H0k1xjH"
        >
          © 
          2023
           
          Tabulous.fr. Tous droits réservés.
        </section>
         
        <a
          class="s-_Wq18H0k1xjH"
          href="/terms-of-service"
        >
          Mentions Légales & Conditions Générales d'Utilisation
        </a>
         
        <a
          class="s-_Wq18H0k1xjH"
          href="#top"
          title="Défiler jusqu'en haut"
        >
          <i
            class="material-icons s-_Wq18H0k1xjH"
          >
            vertical_align_top
          </i>
        </a>
      </footer>
      <!--&lt;PageFooter&gt;-->
    </main>
    <!--&lt;+page&gt;-->
  </div>
</body>
`;

exports[`/login route > expands login panel on error 1`] = `
<body>
  <div>
     
    <main
      class="s-BUJUVtHZo5xi"
    >
      <i
        class="s-BUJUVtHZo5xi"
        id="top"
      />
       
      <header
        class="s-c1MG4BWNDc59"
      >
        <nav
          class="s-c1MG4BWNDc59"
        >
          <button
            class="s-c1MG4BWNDc59"
          >
            <svg
              height="27.035763"
              version="1.1"
              viewBox="0 0 37.126423 27.035763"
              width="37.126423"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:svg="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(-5.7081121,-5.3321023)"
              >
                <path
                  d="m 25.120364,19.713484 a 16.171131,6.3284572 0.15697335 0 0 -10.950753,1.507401 c 0.140483,0.350402 0.240812,0.598413 0.240812,0.598413 l -0.787548,-0.409277 a 16.171131,6.3284572 0.15697335 0 0 -1.884123,0.829407 c 0.463249,0.204263 0.790513,0.438583 0.805119,0.730704 0.03586,0.717292 -1.395039,1.030311 -2.9124841,0.762228 a 16.171131,6.3284572 0.15697335 0 0 -1.0185424,1.698604 16.171131,6.3284572 0.15697335 0 0 14.6461345,6.907589 16.171131,6.3284572 0.15697335 0 0 17.564302,-5.691126 16.171131,6.3284572 0.15697335 0 0 0.0016,-1.133781 c -1.581229,1.267596 -4.142285,2.250232 -4.714441,0.95188 -0.524486,-1.190168 2.444087,-1.664571 3.564123,-2.796728 a 16.171131,6.3284572 0.15697335 0 0 -5.535579,-2.750737 c -0.157951,1.104143 0.2541,3.296302 -1.137915,3.298507 -0.816889,0.0013 -1.122444,-2.159577 -0.807186,-3.771346 a 16.171131,6.3284572 0.15697335 0 0 -6.017204,-0.705383 16.171131,6.3284572 0.15697335 0 0 -1.056266,-0.02635 z m -0.912606,1.601969 h 1.580782 v 1.623674 h -1.580782 z m -5.183146,0.960148 1.791105,0.735873 -1.012858,0.817521 -1.791105,-0.735873 z m 11.035502,0.810287 1.012858,0.817521 -1.791105,0.735872 -1.012858,-0.817521 z m -11.185364,0.681095 1.791106,0.735872 -1.013375,0.817522 -1.791105,-0.735873 z m 15.300875,0.373621 1.012858,0.817521 -1.791105,0.735872 -1.012858,-0.817521 z m -17.849039,0.125057 1.791105,0.735872 -1.012858,0.817521 -1.791106,-0.735872 z"
                  style="font-variation-settings: normal; fill: #000000; fill-opacity: 1; fill-rule: evenodd; stroke: none; stroke-width: 0.634999; stroke-linecap: round; stroke-linejoin: miter; stroke-miterlimit: 4; stroke-dasharray: none; stroke-dashoffset: 0; stroke-opacity: 0.5117; stop-color: rgb(0, 0, 0);"
                />
                <path
                  d="m 35.550623,21.31134 0.668087,-2.021097 c 0.159717,-0.483175 -1.429238,0.453831 -1.991196,1.166561 -0.54224,0.687723 0.350734,3.781871 -1.30307,3.78449 -1.073283,0.0017 -1.264756,-3.730088 -0.323839,-5.043292 0.553159,-0.772023 3.068597,-2.275848 4.221601,-2.288246 1.590592,-0.0171 4.536182,1.713781 5.305128,2.835228 0.562386,0.820194 0.221571,3.922708 -0.437095,4.841158 -1.149572,1.602976 -4.944053,3.520061 -5.656223,1.903992 -0.561409,-1.273959 2.879328,-1.727923 3.771314,-3.046188 0.703797,-1.040142 0.517619,-2.429842 0.517619,-2.429842 m -2.654858,-7.866815 a 2.3234062,2.8584074 14.77332 0 0 1.751252,3.171591 2.3234062,2.8584074 14.77332 0 0 2.844305,-2.433408 2.3234062,2.8584074 14.77332 0 0 -1.751253,-3.171591 2.3234062,2.8584074 14.77332 0 0 -2.844304,2.433408 z"
                  style="font-variation-settings: normal; fill: none; fill-opacity: 1; stroke: #000000; stroke-width: 0.846663; stroke-linecap: round; stroke-linejoin: miter; stroke-miterlimit: 4; stroke-dasharray: none; stroke-dashoffset: 0; stroke-opacity: 1; stop-color: rgb(0, 0, 0);"
                />
                <path
                  d="m 21.69336,20.588834 c 0,0 0.02678,-2.677038 -0.206019,-3.091041 -0.776379,-1.380737 -2.599184,-4.025216 -2.613438,-5.966523 -0.01639,-2.2322799 1.752164,-6.4643449 2.726857,-5.6885019 1.078456,0.858437 -1.085657,4.5175159 -0.381412,6.1633349 0.418802,0.978742 1.010211,2.291873 1.83974,2.745834 1.018025,0.557116 3.010225,0.631654 4.047692,0.111544 1.21019,-0.606701 1.861782,-1.447969 2.336439,-2.879006 0.427503,-1.288873 -1.154588,-4.9427149 -0.07921,-5.3658779 1.462782,-0.575606 2.312798,3.6986019 2.203533,5.6037499 -0.103141,1.798372 -2.31399,4.015614 -2.819254,5.217566 -0.341408,0.812161 -0.156693,3.467564 -0.156693,3.467564 M 27.33896,11.429663 a 2.033196,2.750803 1.8373958 0 1 -2.29222,2.410331 2.033196,2.750803 1.8373958 0 1 -1.737975,-3.052389 2.033196,2.750803 1.8373958 0 1 2.29222,-2.4103309 2.033196,2.750803 1.8373958 0 1 1.737975,3.0523889 z"
                  style="font-variation-settings: normal; fill: none; fill-opacity: 1; stroke: #000000; stroke-width: 0.846663; stroke-linecap: round; stroke-linejoin: miter; stroke-miterlimit: 4; stroke-dasharray: none; stroke-dashoffset: 0; stroke-opacity: 1; stop-color: rgb(0, 0, 0);"
                />
                <path
                  d="m 14.333877,21.844254 c 0,0 -1.220706,-3.003311 -1.32075,-3.465503 -0.05842,-0.269875 1.287063,1.131397 2.623366,0.903835 1.49769,-0.255045 3.519874,-4.235044 1.763283,-4.584697 -0.664466,-0.132263 -1.50618,2.482816 -2.146704,2.604076 -0.595067,0.112655 -2.319707,-2.019375 -3.525543,-2.212273 -2.186247,-0.349735 -3.979353,0.802561 -4.169399,1.046062 -1.192202,1.527537 -1.886942,4.551923 -1.053854,5.856331 1.452582,2.27438 6.026744,2.278506 5.962967,1.002872 -0.0501,-1.002024 -3.762944,-1.331489 -4.117637,-2.056564 -0.192935,-0.394403 0.201973,-2.099557 0.201973,-2.099557 m 2.667282,-7.575614 A 2.8582202,2.3235584 84.877672 0 1 8.9625898,14.098673 2.8582202,2.3235584 84.877672 0 1 6.5644114,11.225541 2.8582202,2.3235584 84.877672 0 1 8.8206822,8.3900889 2.8582202,2.3235584 84.877672 0 1 11.218861,11.263222 Z"
                  style="fill: none; stroke: #000000; stroke-width: 0.846663; stroke-linecap: round; stroke-linejoin: miter; stroke-dasharray: none; stroke-opacity: 1;"
                />
              </g>
            </svg>
            <!--&lt;Tabulous-logo.svg.rollup-plugin&gt;-->
          </button>
           
          <div
            class="s-c1MG4BWNDc59"
          >
            <h1
              class="s-BUJUVtHZo5xi"
            >
              Connexion
            </h1>
          </div>
           
          <span
            class="nav s-c1MG4BWNDc59"
          />
        </nav>
         
        <div
          class="s-c1MG4BWNDc59"
        />
      </header>
      <!--&lt;Header&gt;-->
       
      <section
        class="s-BUJUVtHZo5xi"
      >
        <span
          class="s-j2LTsWktqK_4"
        >
          <div
            class="pane s-j2LTsWktqK_4"
          >
            <h2
              class="s-j2LTsWktqK_4"
            >
              Bienvenue !
            </h2>
            
             
            <div
              class="container s-OEREYjlWzGh2"
            >
               
               
              <details
                class="s-OEREYjlWzGh2"
                open=""
              >
                <summary
                  class="s-OEREYjlWzGh2"
                  title="password-toggle"
                >
                  <span
                    class="material-icons s-OEREYjlWzGh2"
                  >
                    arrow_back
                  </span>
                  Autres options
                </summary>
                 
                <form
                  class="s-OEREYjlWzGh2"
                  method="POST"
                >
                  <div
                    class="row s-OEREYjlWzGh2"
                  >
                    <fieldset
                      class="s-Fe6WNbR-55zp"
                    >
                      <legend
                        class="s-Fe6WNbR-55zp"
                      >
                        Identifiant
                      </legend>
                       
                      <input
                        class="s-Fe6WNbR-55zp"
                        name="id"
                      />
                    </fieldset>
                    <!--&lt;Input&gt;-->
                  </div>
                   
                  <div
                    class="row s-OEREYjlWzGh2"
                  >
                    <fieldset
                      class="s-Fe6WNbR-55zp"
                    >
                      <legend
                        class="s-Fe6WNbR-55zp"
                      >
                        Mot de passe
                      </legend>
                       
                      <input
                        class="s-Fe6WNbR-55zp"
                        data-testid="password"
                        name="password"
                        type="password"
                      />
                    </fieldset>
                    <!--&lt;Input&gt;-->
                  </div>
                   
                  <div
                    class="error s-OEREYjlWzGh2"
                  >
                    Pseudo ou mot de passe incorrect. Merci de réessayer.
                  </div>
                   
                  <div
                    class="actions s-OEREYjlWzGh2"
                  >
                    <button
                      class="primary has-text s-NMyTiX1zi6rz"
                      disabled=""
                      type="submit"
                    >
                      <span
                        class="material-icons s-NMyTiX1zi6rz"
                      >
                        emoji_people
                      </span>
                      
                      <span
                        class="text s-NMyTiX1zi6rz"
                      >
                        Entrer
                      </span>
                      
                      
                    </button>
                    <!--&lt;Button&gt;-->
                  </div>
                   
                  <input
                    class="s-OEREYjlWzGh2"
                    name="redirect"
                    type="hidden"
                    value=""
                  />
                </form>
              </details>
            </div>
          </div>
        </span>
        <!--&lt;Pane&gt;-->
        <!--&lt;Form&gt;-->
      </section>
       
      <footer
        class="s-_Wq18H0k1xjH"
      >
        <section
          class="s-_Wq18H0k1xjH"
        >
          © 
          2023
           
          Tabulous.fr. Tous droits réservés.
        </section>
         
        <a
          class="s-_Wq18H0k1xjH"
          href="/terms-of-service"
        >
          Mentions Légales & Conditions Générales d'Utilisation
        </a>
         
        <a
          class="s-_Wq18H0k1xjH"
          href="#top"
          title="Défiler jusqu'en haut"
        >
          <i
            class="material-icons s-_Wq18H0k1xjH"
          >
            vertical_align_top
          </i>
        </a>
      </footer>
      <!--&lt;PageFooter&gt;-->
    </main>
    <!--&lt;+page&gt;-->
  </div>
</body>
`;
